vue-frontend/src/components/home-new/LandingSection.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <div class="container-fluid">
+    <img :src="landingbackgroundImage" class="background" />
+    <div class="container">
+      <div class="flex-div">
+        <div class="image">
+          <img :src="landingViewImage" alt="landing-view-image" />
+        </div>
+        <div class="flex-detail">
+          <div class="description">
+            <div class="title-div v2-header-3-text">
+              We develop amazing products to help
+              <div class="typewriter">
+                <span class="wrapper">
+                  <p class="v2-canopas-gradient-text fw-bold">Entrepreneurs</p>
+                </span>
+              </div>
+              bring their ideas to life.
+            </div>
+            <div class="mt-5">
+              <a :href="contactURL" class="v2-normal-3-text v2-button">
+                Let's talk
+                <font-awesome-icon
+                  class="arrow fa"
+                  icon="arrow-right"
+                  id="leftArrow"
+                />
+              </a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+<script type="module">
+import landingbackgroundImage from "@/assets/images/Landing/landingbackground.webp";
+import landingViewImage from "@/assets/images/Landing/landingImage.webp";
+import { FontAwesomeIcon } from "@fortawesome/vue-fontawesome";
+
+export default {
+  data() {
+    return {
+      landingbackgroundImage: landingbackgroundImage,
+      landingViewImage: landingViewImage,
+      contactURL: "/contact",
+    };
+  },
+  components: {
+    FontAwesomeIcon,
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.container-fluid {
+  padding-top: 6%;
+  padding-bottom: 15%;
+}
+
+.flex-div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.flex-div > .image {
+  flex: 1 0 0%;
+  text-align: right;
+}
+
+.image > img {
+  vertical-align: top;
+  max-width: 100%;
+  max-height: 100%;
+}
+.flex-div > .flex-detail {
+  flex: 2 0 0%;
+}
+
+.background {
+  width: 100%;
+  object-fit: cover;
+  position: absolute;
+  left: 0;
+  z-index: -5;
+}
+
+.v2-header-3-text {
+  font-weight: 400;
+}
+
+.description {
+  padding-top: 10%;
+  width: 90%;
+  margin: auto;
+}
+
+.title-div {
+  padding-bottom: 10%;
+}
+
+.landing-image {
+  padding-bottom: 5%;
+  width: 80%;
+  height: 80%;
+  object-fit: cover;
+}
+
+.typewriter {
+  display: flex;
+  align-items: center;
+}
+
+.typewriter p {
```

Comment:
Is this CSS working, as the typewriter had no `p` child?

Reply:
Yes, this css work on blink cursor